src/test/java/co/rsk/federate/config/SignerConfigBuilder.java

```diff
@@ -0,0 +1,54 @@
+package co.rsk.federate.config;
+
+import static com.typesafe.config.ConfigValueFactory.fromAnyRef;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import co.rsk.federate.signing.config.SignerConfig;
+
+public class SignerConfigBuilder {
+  private Config config;
+
+  private SignerConfigBuilder() {
+    this.config = ConfigFactory.empty();
+  }
+
+  public static SignerConfigBuilder builder() {
+    return new SignerConfigBuilder();
+  }
```

Comment:
Out of curiosity, any advantage of doing this instead of a public constructor?

Reply:
Not an advantage but is more related to readability and clarity for me. I also used extensively [lombok](https://projectlombok.org/features/Builder) and this is how it implements the builder pattern.